Dockerfile

```diff
@@ -14,7 +14,7 @@ ADD package.json /action/package.json
 ADD package-lock.json /action/package-lock.json
 ADD composer.schema.json /action/
 ADD laminas-ci.schema.json /action/
-RUN (cd /action ; npm install)
+RUN (cd /action ; npm install --no-save)
```

Comment:
Potentially `npm ci`

Reply:
ah, is that an alias? TBH I usually do not use npm so gonna change that.